apps/api/src/app/workflows-v2/shared/step-type-to-control.mapper.ts

```diff
@@ -1,11 +1,12 @@
 import { ActionStepEnum, ChannelStepEnum } from '@novu/framework/internal';
```

Comment:
Nit-picky: This file could be replaced by a mapper returned from the `./schemas/index.ts` to avoid introducing a new filename, mapper.

Reply:
I like the idea and have used this pattern myself, but I noticed that not everyone on the team is aligned with it. I suggest introducing it to the team first.